sms_analysis.ipynb

```diff
@@ -535,30 +622,28 @@
    "execution_count": null,
    "metadata": {
     "collapsed": false,
+    "deletable": true,
+    "editable": true,
     "scrolled": true
    },
    "outputs": [],
    "source": [
     "def get_word_summary_for_contact(contact, top_n=25):\n",
-    "    contact = contact.encode('utf-8')\n",
```

Comment:
The problem appears to be that python 2 and 3 use string differently. You can not call `.encode('utf-8')` in python 3. I've tested removing it with python 2 and 3 without any problems.